### PR TITLE
Implement CCKey class

### PR DESCRIPTION
**Pull request by posgnu**

### CODE REVIEW

File: codechain/keystore/CCkey.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+class CCkey:
+    def __init__(self, db_path):
+        self.platform = keystoreManager(KeyType.PLATFORM, db_path)
+        self.asset = keystoreManager(KeyType.ASSET, db_path)
+        self.db_path = db_path
+
+    @staticmethod
+    def create(cls, **kwargs):
```
Comment (foriequal0) [inline codechain/keystore/CCkey.py]: `@staticmethod` doesn't take `cls`

Comment (posgnu) [reply codechain/keystore/CCkey.py]: I removed it.

File: codechain/keystore/CCkey.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+from jsondb.db import Database
+
+from .KeystoreManager import keystoreManager
+from .KeyType import KeyType
+
+
+class CCkey:
+    def __init__(self, db_path):
+        self.platform = keystoreManager(KeyType.PLATFORM, db_path)
+        self.asset = keystoreManager(KeyType.ASSET, db_path)
```
Comment (foriequal0) [inline codechain/keystore/CCkey.py]: I think this is better design
```
self.db = Database(db_path)
self.db.data(dictionary={"meta": "", "platform": [], "asset": [], "hdwseed": []}) 
self.platform = keystoreManager(KeyType.PLATFORM, self.db)
self.asset = keystoreManager(KeyType.ASSET, self.db)
```
It clearly express that two keystoreManager shares the same data. Also `jsondb` provides same interface with the `dict`, it gives you more flexiblility to test `keystoreManager` without involving file operation.

**Review by foriequal0 — CHANGES_REQUESTED** [review]

File: codechain/keystore/CCkey.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+        return (
+            (meta is not None and meta != "")
+            or (platform is not None and len(platform) != 0)
+            or (asset is not None and len(asset) != 0)
+            or (hdwseed is not None and len(hdwseed) != 0)
+        )
+
+    def get_meta(self):
+        db = Database(self.db_path)
```
Comment (foriequal0) [inline codechain/keystore/CCkey.py]: You can replace this `db = Database(self.db_path)` with `self.db` if you change the design.

**Review by foriequal0 — CHANGES_REQUESTED** [review]

File: codechain/keystore/CCkey.py (hunk 2 above)
Comment (posgnu) [reply codechain/keystore/CCkey.py]: Yes I changed it.

File: codechain/keystore/CCkey.py (hunk 3 above)
Comment (posgnu) [reply codechain/keystore/CCkey.py]: I replaced it.

**Review by foriequal0 — APPROVED** [review]

**Pull request merged**